### ISSUE TITLE
Verify cluster can be created with backup data set

### ISSUE THREAD

**Issue by sopel** [cityindex-attic/logsearch#118]
This details #115 an depends on #116 - it should be possible to start the cluster with a backed up EBS volume snapshot of the data set with a single command by means of creating a stack from the existing CFN template and supplying a resp. parameter.
- a test data set would obviously be helpful, but presumably not required, i.e. we can test with existing ones eventually, if the size doesn't make this unwieldy
  - :question: @mrdavidlaing - knowing how much time is required for a full recovery from backup is also important though, but presumably not a priority right now (if the operation is confirmed as such)?
- :information_source: automatically copying the backup across regions is meanwhile easy, but out of scope here (would be handled via labs-operations in case)
